kofta/src/app/components/Footer.tsx

```diff
@@ -11,7 +11,7 @@ export const Footer: React.FC<FooterProps> = ({ isLogin }) => {
 	const { t } = useTypeSafeTranslation();
 
 	return (
-		<div className={`justify-around flex text-center`}>
+		<div className={`justify-between flex text-center`}>
```

Comment:
will this smash the footer against the side on mobile?

Reply:
No I don't think so. Looks almost identical to how it was before on mobile: below is the resulting screenshot. If you don't like it, I can revert this too no problem.
![Screen Shot 2021-03-06 at 5 54 02 AM](https://user-images.githubusercontent.com/70864612/110209377-209bfd80-7e41-11eb-8993-d7687001b950.png)